src/main/java/de/tum/in/www1/artemis/web/rest/metis/conversation/dtos/ChannelDTO.java

```diff
@@ -40,8 +40,24 @@ public class ChannelDTO extends ConversationDTO {
      */
     private String tutorialGroupTitle;
 
+    // property not taken from entity
+    /**
+     * Determines the subtype of the channel, either "general" , "exercise" or "lecture" depending on whether the channel
+     * is associated with an exercise/lecture or not
+     */
+    private String subType;
```

Comment:
would it make sense to define an enum here?

Reply:
I defined one.